docs/resources/samples/add-excel-comments.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Add comments in Excel'
+description: 'Learn how to use Office Scripts to add comments in a worksheet.'
+ms.date: 03/29/2021
+localization_priority: Normal
+---
+
+# Add comments in Excel
+
+This sample shows how to add comments to a cell including [@mentioning](https://support.microsoft.com/office/90701709-5dc1-41c7-aa48-b01d4a46e8c7) a colleague.
+
+## Example scenario
+
+* The team lead maintains the shift schedule. The team lead assigns an employee ID to the shift record.
+* The team lead wishes to notify the employee. By adding a comment that @mentions the employee, the employee is emailed with a custom message from the sheet.
+* The employee can then address and respond to the comment.
+
+## Solution
+
+1. The Office Script extracts employee information from the employee sheet.
+1. The script then cross references the shift record and adds a comment to the appropriate cell by including the relevant employee email in the team lead's comment.
+1. It also removes any existing comment on the cell before adding a new comment.
+
+## Sample Excel file
+
+Download the file <a href="excel-comments.xlsx">excel-comments.xlsx</a> used in this sample and try it out yourself!
+
+## Office Script
+
+```TypeScript
```

Comment:
Add code comments

Reply:
Yeah, I had received this feedback before. We're tracking with this item: 4958338